portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,19 +14,35 @@
 
 package com.google.sps.servlets;
 
+import com.google.gson.Gson;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  
+  private ArrayList<String> createList() {
+      ArrayList<String> greetings = new ArrayList<>();
+      greetings.add("Hello!");
+      greetings.add("How are you?");
+      greetings.add("Goodbye!");
+      return greetings;
+  }
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
I'm guessing this name was predetermined, but could you change it to something slightly more descriptive?

Reply:
For some reason GitHub isn't showing exactly what you're referencing, are you referring to the name "doGet"? Sorry about that.

Reply:
Yep! I may have messed up commenting too, no worries :)

Reply:
Okay, question though, since its an overriden method would it affect anything if the name is changed?

Reply:
Oh, yep you're completely right. Oversight on my end! LGTM

Reply:
That's okay!